src/passwordValidation.js

```diff
@@ -6,40 +6,40 @@ export const forbiddenPasswords = ["amG84h6yeQ", "mc9Q20pdjH", "jnT6Q2f8U5"];
  * @param {string} password
  * @returns {boolean}
  */
+
+const checkOrder = (password) => {
+  const ascendingNumbers = "123456789";
+  const descendingNumbers = "987654321";
+
+  // The following line extract of numbers in our password and return them as an array of numbers
+  const arrayOfNumbers = password.match(/([0-9])+/g);
+
+  for (let number of arrayOfNumbers) {
+    if (
+      (ascendingNumbers.includes(number) ||
+        descendingNumbers.includes(number)) &&
+      number.length > 1
+    )
+      return false;
+  }
+
+  return true;
+};
 export default function isValidPassword(password = "") {
+  if (forbiddenPasswords.includes(password)) return false;
+
   // The following line ensures, that password is always a string, like the number 128 -> string "128"
-  if(forbiddenPasswords.includes(password)) return false;
   if (typeof password !== "string") password = String(password);
 
-  const myRegex = /^(?=.*[a-z ])(?=.*[0-9])(?=.*[A-Z])[a-zA-Z0-9]{10}$/gm;
-
-  const numberOfCharacters = (password) => {
-    let object = {};
-    for (let character of password){
-      if(object.hasOwnProperty(character)){
-        object[character]++;
-      }else{
-        object[character] = 1;
-      }
-    }
-    return Object.entries(object).length > 3? true : false;
-  }
+  /*  
+  The following regex checks if the password has exactly 10 characters 
+  and also if it includes upper lower cases and number only(no special characters).
+  */
+  if (!/^(?=.*[a-z ])(?=.*[0-9])(?=.*[A-Z])[a-zA-Z0-9]{10}$/gm.test(password))
+    return false;
```

Comment:
- You can also refactor this code

`  if (!/^(?=.*[a-z ])(?=.*[0-9])(?=.*[A-Z])[a-zA-Z0-9]{10}$/gm.test(password))  return false;`

Like this
`  if (!/^(?=.*[a-z0-9])(?=.*[A-Z])[a-zA-Z0-9]{10}$/gm.test(password))  return false;`